### PR TITLE
Added additional logging to track ooc applications on appeal submit

### PR DESCRIPTION
**Pull request by jkumar-moj**
### Change description ###
Added additional logging to track ooc applications on appeal submit

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AppealSubmitHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+    @BeforeEach
+    public void setUp() {
+
+        appealSubmitHandler =
+                new AppealSubmitHandler();
+    }
+
+    @Test
+    void should_call_successfully() {
```
Comment (bradsorour) [inline src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AppealSubmitHandlerTest.java:50]: Could possibly look to use a Parameterized test here for both events.

**Review by bradsorour — DISMISSED** [review]
Approved with minor comment.

**Review by bradsorour — APPROVED** [review]
LGTM

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AppealSubmitHandlerTest.java (hunk 1 above)
Comment (jkumar-moj) [reply src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AppealSubmitHandlerTest.java]: Thanks Brad. Changes done

**Pull request merged**